htdocs/class/xoopsform/formdhtmltextarea.php

```diff
@@ -233,29 +271,52 @@ public function fontArray()
+                            _editor_dialog += '<option value=\''+_color_ele+'\' style=\'background-color:#'+_color_ele+';color:#'+_color_ele+';\'>#'+_color_ele+'</option>';
+                        }
                     }
                 }
-            }
-            _editor_dialog += '</select>';";
-
+                _editor_dialog += '</select>'";
+        if ($GLOBALS['xoopsConfig']['bootstrap'] == true){
+            $fontStr .= " + '</div></div>';";
+        } else {
+            $fontStr .= ";";
+        }
         $fontStr .= 'document.write(_editor_dialog); </script>';
+        
+        if ($GLOBALS['xoopsConfig']['bootstrap'] == true){
+            $styleStr = "<div class='row'><div class='col-md-12'>";
+            $styleStr .= "<div class='btn-group' role='group'>";
+            $styleStr .= "<button type='button' class='btn btn-default btn-sm' onclick='xoopsMakeBold(\"{$hiddentext}\", \"{$textarea_id}\");' title='" . _XOOPS_FORM_ALT_BOLD . "' aria-label='Left Align'><span class='glyphicon glyphicon-bold' aria-hidden='true'></span></button>";
+            $styleStr .= "<button type='button' class='btn btn-default btn-sm' onclick='xoopsMakeItalic(\"{$hiddentext}\", \"{$textarea_id}\");' title='" . _XOOPS_FORM_ALT_ITALIC . "' aria-label='Left Align'><span class='glyphicon glyphicon-italic' aria-hidden='true'></span></button>";
+            $styleStr .= "<button type='button' class='btn btn-default btn-sm' onclick='xoopsMakeUnderline(\"{$hiddentext}\", \"{$textarea_id}\");' title='" . _XOOPS_FORM_ALT_UNDERLINE . "' aria-label='Left Align'>" . _XOOPS_FORM_ALT_UNDERLINE . "</button>";
+            $styleStr .= "<button type='button' class='btn btn-default btn-sm' onclick='xoopsMakeLineThrough(\"{$hiddentext}\", \"{$textarea_id}\");' title='" . _XOOPS_FORM_ALT_LINETHROUGH . "' aria-label='Left Align'>" . _XOOPS_FORM_ALT_LINETHROUGH . "</button>";
+            $styleStr .= "</div>";
+            
+            $alignStr = "<div class='btn-group' role='group'>";
+            $alignStr .= "<button type='button' class='btn btn-default btn-sm' onclick='xoopsMakeLeft(\"{$hiddentext}\", \"{$textarea_id}\");' title='" . _XOOPS_FORM_ALT_LEFT . "' aria-label='Left Align'><span class='glyphicon glyphicon-align-left' aria-hidden='true'></span></button>";
+            $alignStr .= "<button type='button' class='btn btn-default btn-sm' onclick='xoopsMakeCenter(\"{$hiddentext}\", \"{$textarea_id}\");' title='" . _XOOPS_FORM_ALT_CENTER . "' aria-label='Left Align'><span class='glyphicon glyphicon-align-center' aria-hidden='true'></span></button>";
+            $alignStr .= "<button type='button' class='btn btn-default btn-sm' onclick='xoopsMakeRight(\"{$hiddentext}\", \"{$textarea_id}\");' title='" . _XOOPS_FORM_ALT_RIGHT . "' aria-label='Left Align'><span class='glyphicon glyphicon-align-right' aria-hidden='true'></span></button>";
+            
+            $alignStr .= "</div>";            
+        } else {
```

Comment:
__[phpmd]__
* The method fontArray uses an else expression. Else is never necessary and you can simplify the code to work without else.

<!-- Comment by [SideCI](https://sideci.com) -->